### ISSUE TITLE
ONNX Runtime wrong predictions for GradientBoostingClassifier and XGBClassifier

### ISSUE THREAD

**Issue by ScoreMe** [microsoft/onnxruntime#1338]
[gbt_audit_onnx.py.txt](https://github.com/microsoft/onnxruntime/files/3355021/gbt_audit_onnx.py.txt)
[gbt_audit_onnx_result.txt](https://github.com/microsoft/onnxruntime/files/3355041/gbt_audit_onnx_result.txt)

**Describe the bug**
ONNX Runtime provides uncorrect predictions (always class 0) scoring a model exported from scikit-learn.

In particular, a model is trained using scikit-learn and it is scored (using scikit-learn) against the test set. Then, the same model is exported in ONNX format. Afterwards, the model is imported (from ONNX format) and scored against the same test set using ONNX Runtime (through Python API). Finally the predictions provided by scikit-learn and ONNX are compared.

This problem has been encountered using the following scikit-learn models:
- GradientBoostingClassifier
- XGBClassifier

**System information**
- OS Platform and Distribution: Ubuntu 18.04.2 LTS
- ONNX Runtime installed from (source or binary): pip
- ONNX Runtime version: 0.4.0
- Python version: 3.7.3

**To Reproduce**
A script reproducing the aforementioned behavior can be found in attachment "gbt_audit_onnx.py.txt"

**Expected behavior**
Bug fix in ONNX Runtime.
        or
Identification of errors during export/import.

**Screenshots**
![comparison](https://user-images.githubusercontent.com/52496791/60593795-c7461600-9da3-11e9-95ef-c90fcb15055b.png)
![predictions](https://user-images.githubusercontent.com/52496791/60593800-ca410680-9da3-11e9-9f0d-e3bfdca45622.png)

**Comment by xadupre** [comment]
This is the same issue as #1007.
